src/core/utils.ts

```diff
@@ -113,7 +113,13 @@ export function reverseDeepMerge(dest, source = undefined) {
         if (isFunction(src[srcArg])) {
           dest[srcArg] = src[srcArg];
         } else {
-          dest[srcArg] = clone(src[srcArg]);
+          try {
```

Comment:
not sure if this is the right fix as it hide a message that can be useful in some cases